### DISCUSSION TITLE
Getting accent color on windows

### DISCUSSION THREAD

**Discussion by nikhilraojl** [wezterm/wezterm#4786] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

I would like to have a way to get the accent color on windows similar to `wezterm.gui.get_appearance()` and use it to customize scroll bar, pane splits etc.

Currently I am using `io.popen` and run a `reg.exe query <>` in my wezterm config which works but with an huge issue. Whenever I launch wezterm a window(cmd.exe) pops up momentarily and closes, twice in my case 🤷 , before wezterm gui shows up. I have tried minimizing window by default, using `os.execute`, using powershell -windowstyle with no success. There seems to be no easy way to hide a window on launch by default and we have to use something cmdow or another utility. 

Is there a way to extend/expose wezterm's ability to read other registry keys similar to what `wezterm.gui.get_appearance`? or other suggestions?

**Comment by wez** [comment, marked as answer]
Consider using either of these instead of `io.popen`:
* https://wezfurlong.org/wezterm/config/lua/wezterm/run_child_process.html
* https://wezfurlong.org/wezterm/config/lua/wezterm/background_child_process.html

**Reply by nikhilraojl** [reply]
Thank you, this works.
I have another question. Is `.wezter.lua` evaluated twice? 
```lua
local wezterm = require 'wezterm'
local config = {}
if wezterm.config_builder then
	config = wezterm.config_builder()
end
wezterm.log_info("Hello world!")
return config
```
For example if I have the above config, the output on the terminal
```
>> wezterm
20:44:20.504  INFO   logging > lua: Hello world!
20:44:20.652  INFO   logging > lua: Hello world!
```

**Reply by wez** [reply]
Yes, there are a couple of notes about this in some places in the docs:
https://wezfurlong.org/wezterm/config/files.html?h=evaluated+multiple+times#configuration-files

> The configuration file may be evaluated multiple times for each wezterm process both at startup and in response to the configuration file being reloaded. You should avoid taking actions in the main flow of the config file that have side effects; for example, unconditionally launching background processes can result in many of them being spawned over time if you launch many copies of wezterm, or are frequently reloading your config file.

**Reply by nikhilraojl** [reply]
Docs are extensive 🤯, should go through them properly.
Thanks for the help
